packages/expo-video/ios/VideoView.swift

```diff
@@ -6,7 +6,7 @@ import ExpoModulesCore
 public final class VideoView: ExpoView, AVPlayerViewControllerDelegate {
   lazy var playerViewController = AVPlayerViewController()
 
-  var player: VideoPlayer? {
+  weak var player: VideoPlayer? {
```

Comment:
seems fine, but are we not worried about the player deallocating while the view is mounted?

Reply:
The JS side holds the reference as long as the screen is using it. If someone manually deallocates the player too early the video view will reset to an empty player, which can be recovered by setting a new player.